src/routes/Facility.js

```diff
@@ -1,14 +1,14 @@
 import {Router} from 'express'
-import {getFacilities,addFacility} from '../controllers/facilityController';
+import {getFacilities,addFacility,updateFacility,deleteFacility} from '../controllers/facilityController';
 import {_validateFacility} from '../middlewares/FaclityValidation';
-import {req as protection} from '../middlewares/check-auth';
+
 
 const router=Router();
 
 router.get('/',getFacilities);
```

Comment:
Please consider protecting your endpoints, so that only **TRAVEL_ADMIN** will access your endpoints.
Line 8, 26, 55, and 84.

Reply:
i think get facility endpoint doesn't need to be protected because it might be used elsewhere on the web as the whole idea is based on facilities